scribble-lib/scribble/racket.rkt

```diff
@@ -326,6 +326,7 @@
            [first (if escapes?
                       (syntax-case c (code:line)
                         [(code:line e . rest) #'e]
+                      [(code:line . rest) #'rest]
```

Comment:
With the patch which I added, you can now write `(code:line . foo)` which renders as `. foo` (but previously it was not possible).

Also, technically, we could have a first element which is `(code:line (code:line something))`, should the code above be called recursively as long as `code:line` is the first element? (I can amend the patch to include that, if you think it's the right thing to do).